test/test_torch.py

```diff
@@ -363,6 +364,16 @@ def test_remainder(self):
             res2[i, 3] = res2[i, 3] % q
         self.assertEqual(res1, res2)
 
+        # Check the LongTensor case
```

Comment:
Can you add that test case for CUDA too? Also, it would be worth checking that having negative divisors gives correct results too.